h-youjin/LV.0/피자-나눠-먹기(2).js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+  let pizza = 1;
+  while ((pizza * 6) % n) {
+    pizza++;
+  }
+  return pizza;
+}
```

Comment:
```suggestion
function solution(n) {
  let pizza = 1;
  while ((pizza * 6) % n) {
    pizza += 6;
  }
  return pizza;
}
```
- 잘 푸셨네요👍
- 연산 횟수를 줄이기 위해 6씩 더해준다면 효율이 늘어날 것 같습니다!!

Reply:
오호 6씩 더해주는 방법이 연산횟수 측면에서 효율이 더 좋군요 ! 감사합니다 ! :)